web/src/components/pages/profile/info/info.tsx

```diff
@@ -193,6 +184,20 @@ function ProfilePage({
     return null;
   }
 
+  if (!isSaving && isSubmitted && isEnrolled(user.account)) {
+    return (
+      <Redirect
```

Comment:
Did you run into any issues related to "can't perform state update on unmounted components" when you were working on this piece? `Redirect` worked so inconsistently for me. (This seems fine here, just wanted to ask!)

Reply:
Good question - I didn't notice any more than usual, but... that page was already throwing a few similar warnings, so I can't be sure.